test/apps/network/udp_err.c

```diff
@@ -154,6 +154,32 @@ FN_TEST(bind)
 }
 END_TEST()
 
+FN_TEST(bind_reuseaddr)
+{
+	sk_addr.sin_port = htons(8081);
+	struct sockaddr *psaddr = (struct sockaddr *)&sk_addr;
+	socklen_t addrlen = sizeof(sk_addr);
+
+	int enable = 1;
+	int socket1 = CHECK(socket(PF_INET, SOCK_DGRAM | SOCK_NONBLOCK, 0));
```

Comment:
In general, I think we can use more `TEST_SUCC` unless we cannot use it (e.g. in `FN_SETUP`) or the program cannot really continue (e.g. it will definitely abort due to a segmentation error).

For example, even if the `socket()` fails, all later `bind()`/`setsockopt()` will fail because the file descriptor is not valid, but the program won't abort. So I think it is fair to use `TEST_SUCC` instead.